staging/src/k8s.io/kube-aggregator/pkg/apiserver/openapi_aggregator.go

```diff
@@ -250,21 +322,74 @@ func (s *openAPIAggregator) loadApiServiceSpec(handler http.Handler, apiService
 		return nil
```

Comment:
Also here we are completely unprotected by locks, but called from several go-routines.